meteor-app/imports/api/members/members.js

```diff
@@ -1,33 +1,47 @@
-import { Mongo } from 'meteor/mongo';
+import { Mongo } from 'meteor/mongo'
+import { ValidatedMethod } from 'meteor/mdg:validated-method'
+import * as _ from 'lodash'
 
-const Members = new Mongo.Collection('members');
-
-Members.search = function(query){
-  console.log('searching method... ', query)
-  return Members.find({
-    firstname: { $regex: RegExp(query), $options: 'i' }
-  }, {
-    limit: 20
-  })
+class Member {
+  constructor(doc) {
+    _.extend(this, doc)
+    this.name = `${this.firstname} ${this.lastname}`
```

Comment:
This may work, but you are creating work for yourself. I think my suggestion before was to ditch the `firstname` and `lastname` columns in favour of a `name` column. Then all the searching becomes easy.